### ISSUE TITLE
can you share the model performance?

### ISSUE THREAD

**Issue by gaopeng-eugene** [hangzhaomit/semantic-segmentation-pytorch#10]

**Comment by hangzhaomit** [comment]
@gaopeng-eugene 
Thanks for the inquiry, I am still in the process of benchmarking all combinations of the models. Please keep an eye on the repo updates.
Here are the numbers I have now:
resnet34_dilated8 + c1_bilinear: Mean IoU 0.3104, Accuracy: 76.24%
resnet34_dilated8 + psp_bilinear: Mean IoU 0.3558, Accuracy: 77.72%

**Comment by gaopeng-eugene** [comment]
Can you compare the results listed in your paper?
Table 3.
It seems the Accuracy become better and Mean Iou is worse ?

**Comment by gaopeng-eugene** [comment]
And also, the performance you report is on validation right? What's the performance reported in PSPNet paper? I think there is still a large gap, right?

**Comment by hangzhaomit** [comment]
@gaopeng-eugene 
Note that the architectures are different. In my paper, it was based on VGG; PSPNet paper is based on ResNet50, 101, 152, 269.
I have not tune the hyper parameters yet, should further improve after tuning.

**Comment by gaopeng-eugene** [comment]
My setting
4 GPU
8 batch each GPU
default setting
Bn freeze  =  no Bn freeze (Bn freeze is more stable but the upper bound is the same)
resnet34_dilated8 + c1_bilinear


Final Performance on validation set is 79.87 at Epoch 50

**Comment by gaopeng-eugene** [comment]
![](https://hszhao.github.io/projects/pspnet/figures/ADE20K.png)
 Please correct me, the state-of-the-art performance is Pixel Acc 81.69  Mean IoU 44.94

Why the difference between Mean Iou is so big?

**Comment by hangzhaomit** [comment]
@gaopeng-eugene 
Your number looks good, what is your IoU? Please try to use batch size > 16.
Parameter tuning matters a lot, and I will report number on ResNet50 soon.

**Comment by gaopeng-eugene** [comment]
Your program didn't print the IoU. My question is this compairsion 
resnet34_dilated8 + c1_bilinear: Mean IoU 0.3104, Accuracy: 76.24%
PSPNet                                                           0.4494, accuracy  82.69%
There is 6 point difference in Accudacy while 13 point difference in Mean Iou?

SIngle GPU memory is 12 GB which is not enough for 16 batch size per GPU.

**Comment by hangzhaomit** [comment]
@gaopeng-eugene 
But you can evaluate the best model anytime.

Please, they are using PSPNet269 + Pyramid pooling. I am using ResNet34 + 1 Conv.

You can put different parts of your model on different GPUs to get around this easily.

**Comment by gaopeng-eugene** [comment]
Thank you so much for your quick response and clear code.

**Comment by gaopeng-eugene** [comment]
My setting
4 GPU
8 batch each GPU
default setting
Bn freeze = no Bn freeze (Bn freeze is more stable but the upper bound is the same)
resnet34_dilated8 + c1_bilinear

Final Performance on validation set is 79.87 at Epoch 50   （wrong）
Final Performance on validation set is  Acc 76.58   Mean IoU 0.3348 (Right)

Sorry for the mistake

**Comment by hangzhaomit** [comment]
@gaopeng-eugene 
Sounds great, this is better than what I got, closer to PSPNet paper baselines in Table 1.
